cli.js

```diff
@@ -38,6 +38,8 @@ if (isArg('--help') || isArg('-h')) {
 } else if (isArg('--version') || isArg('-v')) {
   process.stdout.write('browserslist ' + pkg.version + '\n')
 } else if (isArg('--update-db')) {
+  process.stdout.write('The --update-db command is deprecated. Please use npx update-browserslist-db@latest instead.\n')
+  process.stdout.write('Browserslist DB update will still be made.\n')
   /* c8 ignore next 3 */
```

Comment:
Can you just move this lines above and increase 3 to 5

Reply:
As you merged it, I won't make any following changes.